typescript/db/migrations/20220318145550_add_uniqueness_labelclass_name/migration.sql

```diff
@@ -0,0 +1,5 @@
+-- Removes duplicates rows based on "datasetId" "name" combination. 
+DELETE FROM "LabelClass" a WHERE EXISTS(SELECT * FROM "LabelClass" b WHERE "a"."datasetId" = "b"."datasetId" AND "a"."name" = "b"."name" AND "b"."id" < "a"."id");
```

Comment:
No, I don't apply migrations before approval.

Reply:
> @nicOwlas This will delete duplicate label-classes _**and**_ the labels linked to them. I don't remember what we said the DSM, is that OK? How much duplicates do we have in production?

In order to take the decision, could you find some examples of those? Ideally if you could export a list with: url of the impacted images, labelClassName, workspaceName.

Can we assign those labels to the labelClass sharing the same name to avoid them being deleted?